server/src/db/catalog/chunk.rs

```diff
@@ -802,23 +859,16 @@ mod tests {
             partition_key,
             mb_chunk,
             ChunkMetrics::new_unregistered(),
-        )
-        .unwrap();
+        );
         assert!(matches!(chunk.stage(), &ChunkStage::Open { .. }));
+    }
 
+    #[test]
+    #[should_panic]
```

Comment:
```suggestion
    #[should_panic(expected = "chunk must not be empty")]
```

Personally I think we should always also check for the panic message so it really hits the error case we intended and not something else (like division by zero).

Reply:
I didn't realise you could do this - nice :ok_hand: